charms/kfp-profile-controller/requirements.txt

```diff
@@ -1,3 +1,5 @@
 ops==1.2.0
 oci-image==1.0.0
 serialized-data-interface<0.4   
+lightkube<0.9
+tenacity<8.1
```

Comment:
Are these used in the charm, or just in the tests?  It looks like just in the tests.  If so, they can go [here](https://github.com/natalian98/kfp-operators/blob/c40f729ed26d509d25c50e8e048d16f96184e884/charms/kfp-profile-controller/tox.ini#L70) instead, as putting them in this `requirements.txt` will ship them with the deployed charm

Reply:
Right! Sorry for my ignorance on tox. I will move those here.